cloudstack/service_offering_constrained_resource.go

```diff
@@ -0,0 +1,270 @@
+package cloudstack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/apache/cloudstack-go/v2/cloudstack"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int32planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+)
+
+var (
+	_ resource.Resource              = &serviceOfferingConstrainedResource{}
+	_ resource.ResourceWithConfigure = &serviceOfferingConstrainedResource{}
+)
+
+func NewserviceOfferingConstrainedResource() resource.Resource {
+	return &serviceOfferingConstrainedResource{}
+}
+
+type serviceOfferingConstrainedResource struct {
+	client *cloudstack.CloudStackClient
+}
+
+func (r *serviceOfferingConstrainedResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: serviceOfferingMergeCommonSchema(map[string]schema.Attribute{
+			"cpu_speed": schema.Int32Attribute{
+				Description: "Speed of CPU in Mhz.  This does not apply to kvm.",
```

Comment:
`This does not apply to kvm.`

I don't think this is correct.

Reply:
I updated the description from here https://github.com/apache/cloudstack/pull/9191/files